src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Views/Shared/_ProviderLayout.cshtml

```diff
@@ -20,5 +20,5 @@
 {
     @RenderSection("Scripts", required: false)
     <script src="libs/accessible-autocomplete/accessible-autocomplete.min.js"></script>
-    <script src="/js/app.js" no-cdn></script>
+    <script src="/js/site.js" no-cdn></script>
```

Comment:
You have renamed to site.js - but kept app.css

Reply:
renamed to site.css